framework/wazuh/core/exception.py

```diff
@@ -89,6 +89,7 @@ class WazuhException(Exception):
                'remediation': 'Experimental features can be enabled in WAZUH_PATH/configuration/api.yaml or '
                               'using API endpoint https://documentation.wazuh.com/current/user-manual/api/reference.html#operation/api.controllers.manager_controller.put_api_config or '
                               'https://documentation.wazuh.com/current/user-manual/api/reference.html#operation/api.controllers.cluster_controller.put_api_config'},
+        1123: {'message': "Error communicating with socket. Query too long, maximum allowed size for queries is 64KB"},
```

Comment:
Maybe use common.py variable for SIZE instead of hardcoded 64 KB

Reply:
Done